Data/Aeson.hs

```diff
@@ -120,6 +122,7 @@ module Data.Aeson
     , (.:!)
     , (.!=)
     , object
+    , embeddedJSON
```

Comment:
To match the other functions I think this function should be placed in `Data.Aeson.Types.FromJSON` with the name `withEmbeddedJSON`.

Reply:
Will do.